analyses/npl/05_station_correlation.md

```diff
@@ -0,0 +1,365 @@
+```python
+from pathlib import Path
+import os
+import sys
+from collections import Counter
+
+import pandas as pd
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+import numpy as np
+from scipy.signal import correlate
+from scipy.interpolate import interp1d
+
+path_mod = f"{Path(os.path.dirname(os.path.realpath(''))).parents[0]}/"
+sys.path.append(path_mod)
+
+from src.indicators.flooding.glofas import utils, glofas
+```
+
+```python
+COUNTRY_ISO3 = 'npl'
+
+DURATION = 1 # Number of days that the GloFAS threshold should be exceeded
+
+RP_LIST = [1.5, 2, 3, 5, 10, 20] # List of return period values to compare
+
+STATIONS = {
+    'Koshi': ['Chatara', 'Simle', 'Majhitar', 'Kampughat'],
+    'Karnali': ['Chisapani', 'Asaraghat', 'Dipayal', 'Samajhighat'],
+    'Rapti': ['Kusum'],
+    'Bagmati': ['Rai_goan'],
+    'Babai': ['Chepang']
+}
+STATIONS_BY_MAJOR_BASIN = {
+    'Koshi': ['Chatara', 'Simle', 'Majhitar', 'Kampughat', 'Rai_goan'],
+    'Karnali': ['Chisapani', 'Asaraghat', 'Dipayal', 'Samajhighat', 'Kusum', 'Chepang'],
+}
+```
+
+```python
+ds_glofas_reanalysis = utils.get_glofas_reanalysis(
```

Comment:
I've encountered and fixed this on another branch that I'm working on, but I think in ```glofas.py``` the ```_get_processed_filepath()``` method needs to be adjusted to add ```public``` to the filepath. It's odd because I'm not sure how this happened...?

Reply:
I think it keeps getting changed back when I merge in develop or something like that. Hopefully after merging this branch it will be fixed!